### DISCUSSION TITLE
How to add/remove custom query (SQL) to Prisma Migration?

### DISCUSSION THREAD

**Discussion by hohoaisan** [prisma/prisma#14544] [Q&A] [answered]
I'm currently investigating full text search feature to work on PostgresSQL, as in the document. Prisma currently does not support defining and creating PostgresSQL search indices. And require us to make a raw query to create it.
https://www.prisma.io/docs/concepts/components/prisma-client/full-text-search#postgresql-1

However, I can't figure out how to add the 'query' to the migration since there's no change in schema.prisma file. Or future changes when the indices is not needed/schema is changed and I have to add another query to remove previous indices.

Please show me how to do it.

**Comment by austincrim** [comment, marked as answer]
Hey @hohoaisan 👋,

Running `npx prisma migrate dev --create-only` will generate an empty `migration.sql` file that you can add your custom SQL to, even if you haven't changed your `schema.prisma` file.

You can read more about customizing migrations [in our docs](https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/customizing-migrations).

**Reply by hohoaisan** [reply]
Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
